hoomd_polymers/sim/simulation.py

```diff
@@ -472,6 +472,30 @@ def run_NVE(self, n_steps):
         self.run(n_steps)
         self.operations.updaters.remove(std_out_logger_printer)
 
+    def run_displacement_cap(self, n_steps, maximum_displacement=1e-3):
+        """ NVE based integrator that Puts a cap on the maximum displacement per time step.
```

Comment:
Lets add a quick note about the idea that this would be a good run method to call initially to relax the system, deal with overlapping particles, etc..